CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 [comment]: <> (All notable changes to this project will be documented in this file.)
 
+# 0.10.0
+### Changes:
+
+- Add `pagination` settings methods `Future<Pagination> getPagination()`, `Future<Task> resetPagination()`, `Future<Task> updatePagination(Pagination pagination)` on `Index` instances.
+- Add `faceting` settings methods `Future<Faceting> getFaceting()`, `Future<Task> resetFaceting()`, `Future<Task> updateFaceting(Faceting faceting)` on `Index` instances.
+- Add `typo tolerance` settings methods `Future<TypoTolerance> getTypoTolerance()`, `Future<Task> resetTypoTolerance()`, `Future<Task> updateTypoTolerance(TypoTolerance typoTolerance)` on `Index` instances.
+- Add filter-builder style:
+  - Added `filterExpression` parameter to the `search` method, which takes a `MeiliOperatorExpressionBase`, you can only use the new parameter or the regular `filter` parameter, but not both. If both are provided, `filter` parameter will take priority.
+  - Added new facade class `Meili` which contains static methods to help create filter expressions.
```

Comment:
`facade` is a specificity of Dart?

Reply:
No, Facade is a design pattern https://refactoring.guru/design-patterns/facade :)